tests/kernel/wave/wave_e2e_test.py

```diff
@@ -43,7 +51,7 @@ def test_copy(shape):
     # elements.
     wave_size = 64
     BLOCK_M = 1
-    BLOCK_N = sympy.Min(N, 256)
+    BLOCK_N = sympy.Max(sympy.Min(N, 256), 64)

```

Comment:
What happens if you remove the max and leave it to min(N, 256)? Just wondering why the max is necessary.

Reply:
I want to keep block size no smaller than `wave_size`, but I should probably change it  `sympy.Max(sympy.Min(N, 256), wave_size)` to make it clear

Reply:
updated

Reply:
thanks